battle/main.py

```diff
@@ -1569,7 +1569,7 @@ def m_woods():
                             steal_right()
                         elif not Rikku.has_overdrive(combat=True):
                             logger.debug("Charging")
-                            CurrentPlayer().attack(target_id=Rikku, direction_hint="u")
+                            CurrentPlayer().attack(Rikku, direction_hint="u")
```

Comment:
Any reason you removed this? It reads better if it's still there

Reply:
Not really, preference more than anything. I will flip it back.